src/test/java/RenameAPlaylist.java

```diff
@@ -0,0 +1,53 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RenameAPlaylist extends BaseTest {
+
+    String email = "holostenco.yuliya@gmail.com";
+    String Password = "te$t$tudent";
+    String xpath = "//ul//li[@class='playlist playlist']";
+    String css ="li>input";
+   String  newPlaylistName=" -Best Club music Selection";
+
+    @Test
+    public void renameSelectedPlaylist(){
+        login();
+        chooseAPlaylist();
+
+wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(css)));
+        WebElement renamedPlaylist=driver.findElement(By.cssSelector(css));
+        renamedPlaylist.sendKeys(newPlaylistName+"\n");
+        Assert.assertTrue(renamedPlaylist.isDisplayed());
+    }
```

Comment:
In test method write the core business logic which is to be tested.

For example

 ```
@Test
  public void renameSelectedPlaylist() {
    login();

    chooseAPlaylist();
    enterNameNewPlaylist();

    Assert.assertTrue(isDisplayedNameNewPlaylist());
  }


  private void enterNameNewPlaylist() {
    wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(css)));
    WebElement renamedPlaylist = driver.findElement(By.cssSelector(css));
    renamedPlaylist.sendKeys(newPlaylistName + "\n");

  }

  private boolean isDisplayedNameNewPlaylist() {
    wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(css)));
    return driver.findElement(By.cssSelector(css)).isDisplayed();

  }
```

![image](https://user-images.githubusercontent.com/104761482/199695537-6bd2c489-103b-42cf-81b9-8634fe4cff4e.png)